domains/didntpot.json

```
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "xDidntPot",
+        "email": "didntpot@protonmail.com"
+      },
+      "record": {
+        "URL": ""https://github.com/xdidntpot""
```

Please remove the extra quotations.

```suggestion
        "URL": "https://github.com/xdidntpot"
```